doc/source/whatsnew/v0.23.1.txt

```diff
@@ -132,4 +132,5 @@ Bug Fixes
 **Other**
 
 - Tab completion on :class:`Index` in IPython no longer outputs deprecation warnings (:issue:`21125`)
+- Timestamp resolution returns a :class:`Timedelta` (in nanoseconds) rather than a normal `timedelta` object (in microseconds) (:issue:`21336`)
```

Comment:
Would you mind moving this entry to the `v0.23.2.txt` whatsnew? The 0.23.1 release is going out today or tomorrow I believe, and it appears the backports have already been established. Thanks for your patience.